pkg/runtime/runtime.go

```diff
@@ -1285,11 +1285,11 @@ func (a *DaprRuntime) createAppChannel() error {
 func (a *DaprRuntime) announceSelf() error {
 	switch a.runtimeConfig.Mode {
 	case modes.StandaloneMode:
-		err := discovery.RegisterMDNS(a.runtimeConfig.ID, a.runtimeConfig.InternalGRPCPort)
+		err := discovery.RegisterMDNS(a.runtimeConfig.ID, []string{a.hostAddress}, a.runtimeConfig.InternalGRPCPort)
```

Comment:
Nit: It looks a bit strange to put this in an array here.  I see that the reason is because `RegisterProxy` needs an array, but I think it would be a bit better to keep that as close to that if possible.  That is, consider passing the `a.hostAddress` as is here, and in `RegisterMDNS`, put it in an array before passing it to `RegisterProxy`.

Reply:
Thanks - makes sense, I was wondering if we might need to support multiple ips at a later point i.e GetHostAddress returns an IPv4 and IPv6 address - is this something we might expect?

Reply:
Yeah, that's a possibility in the future.